services/web/client/source/class/qxapp/component/widget/logger/LoggerView.js

```diff
@@ -57,15 +57,19 @@ Object.freeze(LOG_LEVEL);
 qx.Class.define("qxapp.component.widget.logger.LoggerView", {
   extend: qx.ui.core.Widget,
 
-  construct: function() {
+  construct: function(workbench) {
     this.base();
 
+    this.set({
+      workbench
```

Comment:
odd syntax :) what does this do ?

Reply:
it sets the variable workbench to the property with the same name